mimesis/providers/business.py

```diff
@@ -25,7 +25,11 @@ def company(self) -> str:
         :return: Company name.
 
         :Example:
-            Gamma Systems.
+
+        >>> business = Business()
+        >>> company = business.company()
+        >>> company in business._data['company']['name']
```

Comment:
We need to see a real value here. Like `'Microsoft'`. We can use `random.seed`.

Reply:
https://docs.pytest.org/en/latest/doctest.html#the-doctest-namespace-fixture

Reply:
@sobolevn But what if we update data source? Will result change?

Reply:
I am not sure. Possibly. But in this case it is wise to double check the failing docs. Since it might list the removed data.

Reply:
Okay. So, I'll update all the doctets tonight. Thanks!

Reply:
@sobolevn I have tried to write doctests with seed and as I can judge it's really unreliable solution. So, I'm not sure what else can I do. I'll remove all the doctets to work on them in a different branch. Right now I'll work only on refactoring and improving code, not doctests.